chromadb/test/property/test_add.py

```diff
@@ -287,3 +300,71 @@ def test_add_partial(client: ClientAPI) -> None:
     assert results["ids"] == ["1", "2", "3"]
     assert results["metadatas"] == [{"a": 1}, None, {"a": 3}]
     assert results["documents"] == ["a", "b", None]
+
+
+def test_add_with_no_ids(client: ClientAPI) -> None:
+    """Tests adding a record set with some of the fields set to None."""
```

Comment:
'some of the fields' ?

Reply:
I don't think these belong in property tests - we should remove these and put them elsewhere.